packages/node_modules/brightspace-auth-keys-dynamodb-store/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "brightspace-auth-keys-redis-store",
+  "description": "DynamoDB based public key store implementation for brightspace-auth-keys",
+  "main": "src/public-key-store.js",
+  "scripts": {
+    "test": "nyc --all mocha --timeout 15000 --recursive ./test"
+  },
+  "author": "D2L Corporation",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Brightspace/node-auth-keys.git"
+  },
+  "files": [
+    "src",
+    "README.md",
+    "LICENSE"
+  ],
+  "devDependencies": {
```

Comment:
Can you define these top-level? The peer dep is fine to stay here

Reply:
(The tests failed because I removed the script that runs install within the packages)